clients/src/test/java/org/apache/kafka/clients/producer/KafkaProducerTest.java

```diff
@@ -369,7 +368,7 @@ public void testHeaders() throws Exception {
         MemberModifier.field(KafkaProducer.class, "metadata").set(producer, metadata);
 
         String topic = "topic";
-        Collection<Node> nodes = Collections.singletonList(new Node(0, "host1", 1000));
+        Collections.singletonList(new Node(0, "host1", 1000));
```

Comment:
It seem this can be removed completely? Compare line 375 (I guess, `nodes` was use there at some point but not anymore)